code-challenges.test.js

```diff
@@ -21,9 +21,35 @@ const people = [
 ]
 // Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
 
+describe("sentence", () => {
+  it("returns array sentence with name capitalized", () => {
+    expect(sentence(people)).toEqual(["Ford Perfect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]);
+  })
+})
+
 
 // b) Create the function that makes the test pass.
 
+//declare function
+//takes in an array as parameter
+//iterate over the array using .map
+//concat
+//string interpolation
```

Comment:
I am not seeing these last two steps in your code, but this will definitely get you there.